components/dashboard/src/utils.ts

```diff
@@ -58,8 +58,12 @@ export function isGitpodIo() {
     );
 }
 
+export function isLocalPreview() {
+    return true;
```

Comment:
This will be replaced with `return window.location.hostname === "nip.io` before merging

Reply:
Discussed with @MrSimonEmms, and feeling it would be better to be much more specific here, instead of relying on this.